scripts/reloadModule.sh

```diff
@@ -55,6 +60,8 @@ function reloadModule() {
   local scriptDir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )";
   local moduleDir=$1;
   local omodDir=$moduleDir'/omod/target';
+  local should_reset=$2;
```

Comment:
Do options have to type in a certain order?

Reply:
Yes because this is method parameters.